sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/catalog/interface.scala

```diff
@@ -217,6 +218,7 @@ case class CatalogTable(
     owner: String = "",
     createTime: Long = System.currentTimeMillis,
     lastAccessTime: Long = -1,
+    createVersion: String = "",
```

Comment:
The reason why I did not set the current version by default is that I am afraid some bugs might use the newer version to overwrite the original value that is read from catalog. Thus, I am a little bit conservative here. 

I like your first proposal. `2.2 or prior` looks good to me. It is better than an empty string.